docs/developer-docs/latest/setup-deployment-guides/deployment/hosting-guides/digitalocean-app-platform.md

```diff
@@ -32,6 +32,54 @@ You will configure a database for production. With the setup below, you will onl
 const parse = require('pg-connection-string').parse;
 const config = parse(process.env.DATABASE_URL);
 
+module.exports = () => ({
+  connection: {
+    client: 'postgres',
+    connection: {
+      ...config,
+      ssl: {
+        rejectUnauthorized: false,
+      },
+    },
+  },
+});
+
+```
+
+As the connection object and the `config` returned by parsing the database URL have the same structure, we can use [spread syntax](http://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to copy it.
+
+Alternatively, you can add the values manually:
+
+```javascript
+const parse = require('pg-connection-string').parse;
+const config = parse(process.env.DATABASE_URL);
+
+module.exports = () => ({
+  connection: {
+    client: 'postgres',
+    connection: {
+      host: config.host,
+      port: config.port,
+      database: config.database,
+      user: config.user,
+      password: config.password,
+      ssl: {
+        rejectUnauthorized: false,
+      },
+    },
+  },
+});
+
+```
+
+::: tip
+The configuration above works with Strapi v4. For v3 use the config below.
```

Comment:
Remove the v3 example here, we have dedicated documentation for v3.

Reply:
@ovlb Thank you for your contribution. 
Could you please remove the configuration for v3 and create another PR targetting the `v3` branch to update documentation for Strapi v3 (hosted at docs-v3.strapi.io)?